cmake/BLTMacros.cmake

```diff
@@ -2,6 +2,9 @@
 # other BLT Project Developers. See the top-level COPYRIGHT file for details
 # 
 # SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+# Copyright (c) 2018,2019 Advanced Micro Devices, Inc.
```

Comment:
Please remove this as it is covered in the above copyright and the base COPYRIGHT file.  All contributors retain copyright for their contributions.

Reply:
Hi Chris, we got a go-ahead from our legal department for BLT's new copyright system so I've removed the AMD copyright header from each file as requested.